dss/storage/identifiers.py

```diff
@@ -86,22 +94,28 @@ class FileFQID(ObjectIdentifier):
 
 class TombstoneID(ObjectIdentifier):
 
-    prefix = BUNDLE_PREFIX
+    identity: Type[ObjectIdentifier] = None
```

Comment:
Can you rename this attribute and all similarly named variables to `subject_identity_cls` or `subject_cls`?